hummingbot/connector/exchange/btc_markets/btc_markets_utils.py

```diff
@@ -0,0 +1,87 @@
+from decimal import Decimal
+from typing import Any, Dict
+
+from pydantic import Field, SecretStr
+
+from hummingbot.client.config.config_data_types import BaseConnectorConfigMap, ClientFieldData
+from hummingbot.core.data_type.trade_fee import TradeFeeSchema
+
+CENTRALIZED = True
+EXAMPLE_PAIR = "BTC-AUD"
+
+# Using AUD Market Pair rates, but need to figure out how to determine 30 day AUD vol for rolling
+# https://www.btcmarkets.net/fees
```

Comment:
Just a comment on this comment (as I wrote it originally) is that this is hardcoding fees. As per the link there is a rolling 30 day fee structure where fees decrease as your trade volume increases. This is beneficial as opportunities may increase as fees decrease and the strategy may take this into account when assessing whether to place order. 

If you leave the bot running for days/weeks on end and trade enough to decrease fees, you don't want to have to be editing configuration files to adjust it. I wrote a function earlier in the year to look at this, but can't remember if it was fully functional be might be a starting point if you want to consider including in this PR.

<img width="842" alt="image" src="https://user-images.githubusercontent.com/1315232/197078637-37902c89-65ec-440d-bbb8-1bac70946620.png">

Reply:
Thank you @lt7 

Good point and I agree, it can be improved. I will look into improving it going forward. At this stage I want to get the connector merged first and then can look at doing further improvements.

Regards,

Reply:
@lt7 

I updated the PR to account for dynamic fees calculation also. 

Thx